lib/rails_admin_nestable/helper.rb

```diff
@@ -27,5 +27,23 @@ def action_links(model)
     def tree_max_depth
       @nestable_conf.options[:max_depth] || 'false'
     end
+
+    def object_label(tree_node)
+      custom_object_label = @nestable_conf.options[:object_label]
+      return default_object_label(tree_node) unless custom_object_label.present?
+
+      case custom_object_label
+      when Symbol
+        tree_node.public_send(custom_object_label)
+      when proc { custom_object_label.respond_to? :call }
+        custom_object_label.call(tree_node)
+      else
+        fail 'object_label must be a Symbol or a Proc'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.